class/defaults.yml

```diff
@@ -39,7 +39,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.33.0
+        tag: redis/sla_report
```

Comment:
I'm going to fix that once https://github.com/vshn/appcat/pull/86 is merged and rolled out